frontend/src/data/Breakdowns.ts

```diff
@@ -149,6 +161,18 @@ export class Breakdowns {
     return this.addBreakdown("date");
   }
 
+  demographicBreakdownCount() {
+    let demographicBreakdownCount = 0;
+    [this.age, this.sex, this.race, this.race_nonstandard].forEach(
+      (demographic) => {
+        if (demographic) {
+          demographicBreakdownCount++;
+        }
+      }
+    );
+    return demographicBreakdownCount;
```

Comment:
you can change this whole block to:
`[this.age, this.sex, this.race, this.race_nonstandard].filter((demo) => !!demo).length`